config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -123,6 +123,47 @@ periodics:
             exit $rc
           securityContext:
             privileged: true
+    - name: periodic-integration-cli-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+              preset-build-docker: "true"
+      cron: 0 14 * * *
+      decorate: true
+      decoration_config:
+        timeout: 3h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:f93b6d36ad414d30f584c6555d8c2116de58bd56601da36f4e13d4b5b624114a
+          resources:
+            requests:
+              cpu: "4000m"
+            limits:
+              cpu: "4000m"
```

Comment:
Hope you have test run this job in your development cluster and visualized the view of test results in junit tab?